src/predictor.h

```diff
@@ -23,6 +23,10 @@ class Predictor : public Processor {
  private:
   enum Action { kUnspecified, kSelect, kDelete };
   Action last_action_ = kUnspecified;
+  int page_size_;          // page_size from schema
```

Comment:
No need to read `page_size_` from schema. Just let user set a value equals to schema's page size.